conf/README.md

```diff
@@ -0,0 +1,123 @@
+# Config directory
+
+In this directory we store all the configuration for clusters and OCSCI config
+files.
+
+During the execution, via
+[ocsci pytest plugin](../pytest_customization/ocscilib.py), we are loading all
```

Comment:
this statement needs to be rewritten, probably " During the execution we are loading the config file with different sections (Defaults, RUN etc) and using the values defined in the config file to render yaml with jinja2 templates " ?